### DISCUSSION TITLE
Why do I need to use Apple's genuine charge?

### DISCUSSION THREAD

**Discussion by shahin365** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1011] [General]
I was reading a thread about the AlDente on Reddit and someone shared an email from the developer that said in order for AlDente to work properly you need to use Apple's genuine charge (for MacBook). Is that true? can you elaborate on that and the reason behind it?

**Comment by MatthiasKerbl** [comment]
Hi @shahin365,

In general, Apple's original chargers always work and usually, high-quality third-party chargers which support MacBooks work without any issues as well. However, there might be some issues with low-quality, unsupported chargers. Therefore, when we are troubleshooting issues, one of the first steps we usually recommend is to use the original charger from Apple.

Best,
Matthias

**Reply by shahin365** [reply]
so it's not a requirement for the Aldente feature to work? (like stop charging at the desired percentage, charging from the power adaptor instead of the battery, sailing mode or etc)?

**Reply by MatthiasKerbl** [reply]
Exactly
